src/draw.typ

```diff
@@ -492,13 +492,13 @@
 /// - top-right: Top right
 /// - bottom-left: Bottom left
 /// - bottom-left: Bottom right
-/// - top: Mid between top-left and top-right
-/// - left: Mid between top-left and bottom-left
-/// - right: Mid between top-right and bottom-right
-/// - bottom: Mid between bottom-left and bottom-right
+/// - top: Mid between top left and top right
+/// - left: Mid between top left and bottom left
+/// - right: Mid between top right and bottom right
+/// - bottom: Mid between bottom left and bottom right
 ///
-/// - a (coordinate): Bottom-Left coordinate
-/// - b (coordinate): Top-Right coordinate
+/// - a (coordinate): Bottom left coordinate
+/// - b (coordinate): Top right coordinate
```

Comment:
> The second change is def. incorrect, as it refers to the anchor names, which are hyphenated.

You mean all this? I've noticed that in all other places, dashed words are enclosed in "\``". So if some of these words are referring to the Typst value/argument, then they also must be enclosed with "``", again, to make documentation more consistent.